### DISCUSSION TITLE
Could day js work as a drop in replacement for moment?

### DISCUSSION THREAD

**Discussion by SSDK-Hello** [team-forms/community#11] [Ideas]
Could be an easy if [performance](https://day.js.org/) win if specific features are not used

**Comment by Erin-Dwyer** [comment]
Hi @SSDK-Hello,

Thanks for the suggestion, we have raised this internally as a potential improvement. However it may be tricky if existing customers are using specific features in moment.js which are not supported. We do try to always provide backwards compatability for exisitng forms in production.
